bridge/contracts/AllowTokens.sol

```diff
@@ -153,7 +153,7 @@ contract AllowTokens is Initializable, UpgradableOwnable, UpgradableSecondary, I
 
     function setToken(address token, uint256 typeId) override public notNull(token) {
         require(isOwner() || _msgSender() == primary(), "AllowTokens: unauthorized sender");
-        require(typeId < typeDescriptions.length, "AllowTokens: typeId does not exist");
+        require(typeId < typeDescriptions.length, "AllowTokens: typeId not exist");
```

Comment:
I think it was correct in the previous form

Reply:
32+ characters